package.json

```diff
@@ -6,29 +6,21 @@
   "module": "lib-esm/index.js",
   "exports": {
     ".": {
-      "import": "./lib-esm/index.js",
-      "require": "./lib/index.js"
+      "import": "./lib-esm/index.js"
     },
     "./experimental": {
-      "import": "./lib-esm/experimental/index.js",
-      "require": "./lib/experimental/index.js"
+      "import": "./lib-esm/experimental/index.js"
     },
     "./drafts": {
-      "import": "./lib-esm/drafts/index.js",
-      "require": "./lib/drafts/index.js"
+      "import": "./lib-esm/drafts/index.js"
     },
     "./deprecated": {
-      "import": "./lib-esm/deprecated/index.js",
-      "require": "./lib/deprecated/index.js"
+      "import": "./lib-esm/deprecated/index.js"
     },
     "./lib-esm/*": {
       "import": [
         "./lib-esm/*.js",
```

Comment:
I think if we add `type: module` for the `package.json` that we can remove the `"import"` for each of these and just set the value of each entrypoint to the path. Let me know if this doesn't work!